packages/eslint-plugin/package.json

```diff
@@ -17,6 +17,13 @@
 	"bugs": {
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
+	"files": [
+		"configs",
+		"docs",
```

Comment:
I wouldn't consider `docs` necessary for distributable (it's not relevant for runtime execution).

Reply:
Yes, agreed. I will remove.

Reply:
https://github.com/WordPress/gutenberg/pull/15879/commits/8b48a1376c2dc88f88e0c5e9091597fc18d0b9e0 - fixed.